src/routes/auth.js

```diff
@@ -75,7 +86,8 @@ router.post(
 		} = req.body
 
 
-		const { token, userId } = await userSevice.createNewUser([
+		const { accessToken, refreshToken, userId } = 
```

Comment:
no need to destructure, use one variable and pass it to the transform

Reply:
yes please